src/modules/dashboard/components/FeaturedChannels.tsx

```diff
@@ -47,66 +30,18 @@ const FeaturedChannels: FC<FeaturedChannelsProps> = () => {
       backgroundColor={{ light: 'darkWhite', dark: 'gray-900' }}
       gap={{ ml: 's4', initial: 's2' }}
     >
-      <Box
-        display="flex"
-        justifyContent="space-between"
-        flexDirection={{ tb: 'column' }}
-        gap={{ tb: 's3' }}
-      >
-        <Text
-          variant="h4-bold"
-          color={{ light: 'black', dark: 'white' }}
-        >
-          Featured Notification Channels
-        </Text>
-
-        <Box
-          display="flex"
-          flexDirection="row"
-          alignItems="center"
-          gap="s4"
-        >
-          <Link
-            to="/channels"
-            textProps={{
-              variant: 'h5-semibold',
-              color: { light: 'black', dark: 'white' },
-            }}
-          >
-            View All
-          </Link>
-
-          <Box
-            display="flex"
-            flexDirection="row"
-          >
-            <HoverableSVG
-              onClick={handlePrevious}
-              defaultColor={{ light: 'gray-900', dark: 'gray-400' }}
-              disabled={currentIndex === 0}
-              icon={<PrevIconSlider size={24} />}
-            ></HoverableSVG>
-            <HoverableSVG
-              onClick={handleClick}
-              defaultColor={{ light: 'gray-900', dark: 'gray-400' }}
-              disabled={currentIndex + itemsPerPage >= applicableFeaturedChannels.length}
-              icon={<NextIconSlider size={24} />}
-            ></HoverableSVG>
-          </Box>
-        </Box>
-      </Box>
 
       {showMobileAndTabletView ? (
         <FeaturedChannelsMobileViewList
-          listRef={listRef}
           featuredChannelsList={mobileFeaturedChannelsList}
         />
+
       ) : (
         <FeaturedChannelsList
-          listRef={listRef}
           featuredChannelsList={featureChannelsForCurrrentEnv}
         />
       )}
+
```

Comment:
**UI Broken for tablet view**

<img width="1132" alt="Screenshot 2024-07-08 at 5 13 41 PM" src="https://github.com/push-protocol/push-dapp/assets/25508878/3f88d053-12fa-4d0d-b423-ed26bfdcf0c4">



**For mobile small the spacing is off on the right side.**

<img width="624" alt="Screenshot 2024-07-08 at 5 13 51 PM" src="https://github.com/push-protocol/push-dapp/assets/25508878/10a8d577-e675-4677-8f8e-83622f288938">



**For mobile medium the spacing is off on the right side.**

<img width="650" alt="Screenshot 2024-07-08 at 5 13 57 PM" src="https://github.com/push-protocol/push-dapp/assets/25508878/8668f6a2-2828-441a-be70-8693f2f5b8c7">



**For mobile large the spacing is off on the right side.**

<img width="858" alt="Screenshot 2024-07-08 at 5 14 02 PM" src="https://github.com/push-protocol/push-dapp/assets/25508878/6e9c5805-7821-4e4d-b9be-75f3a6be7aff">



**For laptop the spacing is off on the right side and item width doesn't matches the design**

<img width="1504" alt="Screenshot 2024-07-08 at 5 14 11 PM" src="https://github.com/push-protocol/push-dapp/assets/25508878/f88c153a-2dd9-4850-a0ae-541862a7223e">



**For desktop the spacing is off on the right side and the width doesn't matches the design**

<img width="2559" alt="Screenshot 2024-07-08 at 5 14 27 PM" src="https://github.com/push-protocol/push-dapp/assets/25508878/11ee6dd7-de80-408a-9cb4-93118e669608">